### PR TITLE
components refactored to use reusable variables to define styles

### PR DESCRIPTION
**Pull request by asiedu-george**

### CODE REVIEW

File: src/app/landing-page/discover/discover.component.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        <p>
+            Explore our curated listings to find homes that perfectly match your preferences and lifestyle. 
+            Whether you're looking for a cozy apartment in the city or a spacious house in the suburbs, <br>
+            we have something for everyone. Start your journey towards finding your dream home today.
+        </p>
+        <button routerLink="/properties" type="button" class="btn">
+            View Properties
+            <img src="black-right-arrow.svg" alt="right-arrow">
+        </button>
+    </div>
```
Comment (Kwabena-Owusu-Achiaw) [inline src/app/landing-page/discover/discover.component.html:16]: Unicode will be a much preferred option to using punctuations directly

File: src/app/shared/back-to-top/back-to-top.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<span class="top-icon" (click)="scrollToTop()">
```
Comment (Kwabena-Owusu-Achiaw) [inline src/app/shared/back-to-top/back-to-top.component.html]: a <button> tag will be a much preferred  option

File: src/app/shared/back-to-top/back-to-top.component.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+@Component({
+  selector: 'app-back-to-top',
+  templateUrl: './back-to-top.component.html',
+  styleUrl: './back-to-top.component.scss'
+})
+export class BackToTopComponent {
+  @HostListener('window:scroll', [])
+  onWindowScroll() {
+    const scrollButton = document.querySelector('.top-icon') as HTMLElement
+    if (document.body.scrollTop > 200 || document.documentElement.scrollTop > 200) {
```
Comment (Kwabena-Owusu-Achiaw) [inline src/app/shared/back-to-top/back-to-top.component.ts]: Defining  what the 200 value stands for will be a preferred approach which can be used through out the component.

File: src/app/shared/back-to-top/back-to-top.component.html [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<button class="top-icon" (click)="scrollToTop()">
+    <img src="up-arrow.svg" alt="up-arrow">
+</button>
```
Comment (george-asiedu) [inline src/app/shared/back-to-top/back-to-top.component.html:3]: refactor done

File: src/app/shared/back-to-top/back-to-top.component.ts (hunk 3 above)
Comment (george-asiedu) [reply src/app/shared/back-to-top/back-to-top.component.ts]: refactor done

**Review by Kwabena-Owusu-Achiaw — APPROVED** [review]

File: src/app/landing-page/discover/discover.component.html (hunk 1 above)
Comment (asiedu-george) [reply src/app/landing-page/discover/discover.component.html:16]: done

File: src/app/shared/back-to-top/back-to-top.component.html (hunk 2 above)
Comment (asiedu-george) [reply src/app/shared/back-to-top/back-to-top.component.html]: done

**Pull request merged**